e2e/cypress/integration/post_header/post_header_spec.js

```diff
@@ -33,8 +35,11 @@ describe('Post Header', () => {
             // # Click timestamp of a post
             cy.clickPostTime(postId);
 
-            // * Check that it redirected to permalink URL
-            cy.url().should('include', `/ad-1/pl/${postId}`);
+            // * Check if url include the permalink
+            cy.url().should('include', `/ad-1/channels/town-square/${postId}`);
+
+            // * Check if url redirects back to parent path eventually
+            cy.wait(TIMEOUTS.SMALL).url().should('include', '/ad-1/channels/town-square').and('not.include', `/${postId}`);
```

Comment:
Succeeding steps failed.  Looks like no longer applicable?
![Screen Shot 2020-03-13 at 10 00 30 AM](https://user-images.githubusercontent.com/5334504/76582566-91038d80-6511-11ea-96fe-2b4ad30eb6d1.png)

Reply:
This failure is known and listed in the audit I posted in the chatroom. Will address in a separate PR